src/state/cached_state.rs

```diff
@@ -25,7 +25,7 @@ pub const UNINITIALIZED_CLASS_HASH: &ClassHash = b"\x00\x00\x00\x00\x00\x00\x00\
 
 #[derive(Default, Clone, Debug, Eq, Getters, MutGetters, PartialEq)]
 pub struct CachedState<T: StateReader> {
-    #[get = "pub"]
+    #[getset(get = "pub", get_mut = "pub")]
```

Comment:
avoid using a getter for this field. Also get_mut is not needed. You can just create a new one.